### PR TITLE
fix: fix vulnerabilities

### PR DESCRIPTION
**Pull request by astencel-sumo**
See the individual commits for details.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -111,7 +111,18 @@ RUN apt-get update \
         jq \
  && gem cleanup \
  && rm -rf /var/lib/apt/lists/ \
- && rm -rf /var/lib/dpkg/info/
+ && rm -rf /var/lib/dpkg/info/ \
```
Comment (swiatekm-sumo) [inline Dockerfile:114]: Can you add a comment explaining why we're doing this?

File: alpine.Dockerfile [hunk 2]
```diff
@@ -104,7 +104,18 @@ RUN apk update \
  && apk add --no-cache \
         ca-certificates \
         snappy-dev \
-        tini
+        tini \
```
Comment (swiatekm-sumo) [inline alpine.Dockerfile:107]: Same thing here.

**Review by swiatekm-sumo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (astencel-sumo) [reply Dockerfile:114]: The comment is in the commit message, isn't this enough?

Comment (astencel-sumo) [reply Dockerfile:118]: OK, added a comment.

File: alpine.Dockerfile (hunk 2 above)
Comment (astencel-sumo) [reply alpine.Dockerfile:110]: Added comment describing why we're doing this.

**Pull request merged**